src/components/Card/Activity.js

```diff
@@ -15,10 +15,15 @@ export default function Activity({ activity, deleteActivity }) {
             : ' hours'}
           )
         </SpanStyled>
-        <BtnDelete onClick={() => deleteActivity(activity)}>&times;</BtnDelete>
+        <BtnDelete onClick={handleDelete}>&times;</BtnDelete>
       </NameStyled>
     </ListStyled>
   )
+
+  function handleDelete(event) {
+    event.stopPropagation()
```

Comment:
Why do you stop the propagation of the event? I don't understand why this is necessary.

Reply:
This is to stop the toggle propagation, if there's not stopPropagation, then when I delete an activity it also toggle the card.